docs/user_guide/sorting_policies.md

```diff
@@ -146,6 +146,33 @@ Resources of the same type are compared for the nodes.
 This results in a node with the highest utilisation to be considered first for assigning new allocation.
 Resulting in a high(er) utilisation of a small(er) number of nodes, better suited for cloud deployments.   
 
+## Resource weighting
+Node sorting policies may use the utilization of a node to determine ordering. Because nodes can have several unique
+resource types, a node's utilization is determined by a weighted average of its individual resource types. Resource
+weighting can be customized by using the `resourceweights` section of `nodesortpolicy`. If `resourceweights` is not
+present or empty, the default configuration sets the weight of both `vcore` and `memory` equally to `1.0`. All other
+resource types are ignored. Only resource types explicitly mentioned will have a weight.
```

Comment:
I think we need to add a note here, users might be confused about "vcore". We can say something like:
> YuniKorn tracks CPU resources as "vcores" internally, this is mapped to the K8s resource type "cpu". For any other resource types in K8s, the naming is consistent in YuniKorn.

Reply:
Updated.